### ISSUE TITLE
mintupdate-cli erroneously updates blacklisted packages

### ISSUE THREAD

**Issue by Martin-Laclaustra** [linuxmint/mintupdate#723]
**Describe the bug**
mintupdate-cli erroneously updates blacklisted packages.

**To Reproduce**
1. Go to mintupdate GUI
2. Blacklist a package (i.e. Thunderbird)
2. Open the terminal
3. Run:
```
mintupdate-cli -r list
# does not list blacklisted packages
sudo mintupdate-cli -r upgrade
# erroneously upgrades blacklisted packages!!
```

**Expected behavior**
The same behavior as button "install updates" in the GUI. I.e. blacklisted packages are not upgraded.

**Distribution:**
- [X] Linux Mint
- [ ] LMDE
- [ ] Other (please specify)

**Software version:**
5.8.1
(freshly installed Mint 20.3, and only the mintupdate GUI upgrade applied)

**Additional context**
Tested with Thunderbird just after fresh install.

**Comment by Sahil-Poonia** [comment]
The bug exists in LMDE Beta 5 too.

I downgraded thunderbird using the following command:
```
sudo apt install thunderbird=1:78.14.0-1~deb11u1
```
the package is already blacklisted through GUI. still, `mintupdate-cli` upgraded thunderbird.

#### Possible reason:
`mintupdate-cli` ignores packages that are either specified with the flag `-i` or `--ignore` or the package is listed in the file `/etc/mintupdate.blacklist`. but the GUI doesn't add the package in this file when blacklisted. in my case, this file doesn't even exist.

#### Possible fix:
GUI should append/add the package to the file `/etc/mintupdate.blacklist` when blacklisted.

**Comment by Sahil-Poonia** [comment]
to prevent thunderbird from upgrading through CLI, use following command:
```
sudo mintupdate-cli -r upgrade -i thunderbird
```

**Comment by Martin-Laclaustra** [comment]
> to prevent thunderbird from upgrading through CLI, use following command:
> 
> ```
> sudo mintupdate-cli -r upgrade -i thunderbird
> ```

Does this blacklist it also for the GUI? If not... that is also needed for coherent functionality.

**Comment by Sahil-Poonia** [comment]
> > to prevent thunderbird from upgrading through CLI, use following command:
> > ```
> > sudo mintupdate-cli -r upgrade -i thunderbird
> > ```
> 
> Does this blacklist it also for the GUI? If not... that is also needed for coherent functionality.

No, it's temporary argument only valid for that particular runtime. I mean, you'll need to specify `-i thunderbird` for every future execution. `-i` flag doesn't add the specified packages to any file.

**Comment by Sahil-Poonia** [comment]
There is a button in the preferences of mintupdate GUI that says "Export blacklist to /etc/mintupdate.blacklist". but for some reason it doesn't quite do that. it will export the blacklist to `/tmp/mintUpdate/blacklist` instead of `/etc/mintupdate.blacklist`.

Instead use `apt-mark`. it's a better option if you want to prevent some packages from upgrading.
